admin-ui/app/index.html

```diff
@@ -98,7 +98,7 @@
                   <span class="pficon pficon-warning-triangle"></span>
                   <span class="pficon pficon-warning-exclamation"></span>
               </span>
-              <a href ng-link="appDetail({app: warning.applicationID, tab: 'activity'})"><strong>Push failed:</strong> {{ warning.applicationName }}</a>
+              <a href ng-link="appDetail({app: warning.applicationID, tab: 'activity'})">Variant <strong>{{ warning.variant.name }}</strong> has push failures.</a>
```

Comment:
I can not tell from there if there on or more errors. I could write `failure(s)` ?